api/datastorage-api/src/main/java/com/cognifide/aet/vs/MetadataDAO.java

```diff
@@ -100,4 +100,10 @@
    * @return list of projects in as DBKeys
    */
   Collection<DBKey> getProjects(String company) throws StorageException;
+
+  /**
+   * @param dbKey - key with project and company name
+   * @return true when database for given dbKey exists
```

Comment:
It should probably return `true` also when the database does not exist but the `allowAutoCreate` flag is set to true :) Please update javadoc